marker_map: { '2916' : [[1,0,0,0.032],[0,1,0,0.043],[0,0,1,1.84],[0,0,0,1]],
              '9334' : [[1,0,0,0.090],[0,1,0,0.080],[0,0,1,1.84],[0,0,0,1]]}